css/components/badges.css

```diff
@@ -181,8 +178,7 @@
 }
```

Comment:
If you are only changing scale with transform for all badge sizes, why not make code shorter by bringing all common styles in a seperate class?